Makefile

```diff
@@ -128,8 +106,7 @@ clean-olena:
 
 # Checkout OCR-D/assets submodule to ./repo/assets
 repo/assets: assets-update
-	git submodule init
-	git submodule update
+	git submodule update --init $@
```

Comment:
```suggestion
	git submodule sync $@
	git submodule update --init $@
```

Reply:
Again, upstream will always be `https://github.com/OCR-D/assets`?

Reply:
> will always be

Likely, but not necessarily. For example, it might be split in the future to reduce the toll for individual repos. And for the olena submodule – it just _did_ change. 

When these changes eventually do happen, going back and forth the history is dangerous/cumbersome without `sync` already in place.